controllers/auth.go

```diff
@@ -463,6 +464,23 @@ func (c *ApiController) Login() {
 			}
 
 			password := authForm.Password
+
+			var organization *object.Organization
+			organization, err = object.GetOrganization(util.GetId("admin", authForm.Organization))
+			if err != nil {
+				c.ResponseError(c.T(err.Error()))
+			}
+			if organization != nil {
```

Comment:
use `err != nil || organization == nil ` in previous if can escape this if statement, and we have get organization in `CheckPassword` function in objects/check.go line 235 , don't get org twice